doc/source/reference/c-api/coremath.rst

```diff
@@ -249,9 +249,14 @@ Linking against the core math library in an extension
 To use the core math library in your own extension, you need to add the npymath
 compile and link options to your extension in your setup.py:
 
+        .. hidden in a comment so as to be included in refguide but not rendered documentation
+          >>> import numpy.distutils.misc_util
+          >>> config = np.distutils.misc_util.Configuration(None, '', '.')
+          >>> with open('foo.c', 'w') as f: pass
```

Comment:
Could use the following
```suggestion
                >>> import numpy.distutils.misc_util
                >>> config = np.distutils.misc_util.Configuration(None, '', '.')
                >>> with open('foo.c', 'w') as f: pass
```
which seems to fit the 8-space indents used by this file. Not a big deal though.